flow-typed/types/text-config.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import {TextStyle} from '../../src/track/text-style';
+
+declare type PKTextConfigObject = {
+  enableCEA708Captions: boolean,
+  useNativeTextTrack: boolean,
+  textTrackDisplaySetting: Object,
+  textStyle: TextStyle | Object,
```

Comment:
why Object? we should be strict here. If you want partial then use [$Shape](https://flow.org/en/docs/types/utilities/#toc-shape)

Reply:
It's not partial, we wanted an option to pass config and create TextStyle instance on the fly, the config is the properties of the object.

Reply:
Ok, but it should be defined as TextStyle or object with list of keys of TextStyle instead of an open object.